VikingQuiz.UI/src/Components/ForgotPasswordComponent/SendEmailComponent/SendEmailComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import axios from 'node_modules/axios';
+import HomeButton from 'src/Components/Buttons/HomeButton/HomeButton';
+import FormComponent from 'src/Components/FormComponent/FormComponent';
+import InputData from 'src/entities/InputData';
+import './SendEmailComponent.css';
+
+class SendEmailComponent extends React.Component<any, any>
+{
+    constructor(props: any) {
+        super(props);
+
+        
+        this.state = {
+            showStatusMessage: false,
+            statusClass: '',
+            statusMessage: '' // the message to show after we call the server to send resend link
```

Comment:
Please remove these kind of comments. A variable shouldn`t require a comment for one to know its meaning.